src/plugins/legacy/itkFilters/itkFiltersToolBox.cpp

```diff
@@ -276,30 +276,35 @@ itkFiltersToolBox::itkFiltersToolBox(QWidget *parent)
     d->thresholdFilterWidget = new QWidget(this);
 
     d->thresholdFilterValue = new medDoubleParameterL( tr("Value"), this);
-    d->thresholdFilterValue->setRange ( std::numeric_limits<double>::lowest() , std::numeric_limits<double>::max() );
-    d->thresholdFilterValue->setValue ( itkFiltersThresholdingProcess::defaultThreshold );
-    d->thresholdFilterValue->setObjectName("thresholdValue");
+    d->thresholdFilterValue->setToolTip(tr("Only keep values according to the threshold type"));
+    d->thresholdFilterValue->setValue(itkFiltersThresholdingProcess::defaultThreshold);
+    d->thresholdFilterValue->setRange(-10000, 10000);
```

Comment:
Pourquoi ne pas garder: `std::numeric_limits<double>::lowest() , std::numeric_limits<double>::max()` qui est algorithmiquement très bien

Reply:
I forgot to explain that these numeric limits values were not allowed in these parameters. They were blocking the range, and we couldn't use the sliders

Reply:
ok